routers/repo/issue.go

```diff
@@ -61,6 +61,23 @@ var (
 	}
 )
 
+// MustAllowUserComment checks to make sure if an issue is locked.
+// If locked and user has permissions to write to the repository,
```

Comment:
What do you think about making it an option whether or not a repo writer can still comment on a locked issue?

Reply:
Sounds nice.

Will this potentially be a global config or an option set at the point of ___locking the issue___ ?

Reply:
I'd make a global option (in the config file) and then a setting in the repo to overwrite the global default.

Reply:
So, what about this? I think it'd be ok if we move the configuration to another pr.

Reply:
@kolaente this PR is already long history, let's move such option to other PR

Reply:
@lafriks sounds good!